src/components/ItemTable/ItemAvailability.tsx

```diff
@@ -87,7 +87,13 @@ const ItemAvailability = ({ item }: ItemAvailabilityProps) => {
         <Button
           id="contact-librarian"
           buttonType="link"
-          sx={{ display: "inline", fontWeight: "inherit", fontSize: "inherit" }}
+          sx={{
```

Comment:
can you move this to a css module?

Reply:
Tried moving this into a CSS module but the specificity wasn't high enough to override the DS styles.